ding_staff.info

```diff
@@ -61,5 +63,7 @@ features[page_manager_pages][] = ding_staff_library
 features[profile2_type][] = ding_staff_profile
 features[rules_config][] = rules_role_staff_assign
 features[taxonomy][] = ding_staff_department
+features[variable][] = realname_pattern
 features[views_view][] = ding_staff
 features[views_view][] = ding_staff_admin_users
+mtime = 1426511413
```

Comment:
This mtime should not be part of the info file. It's from meta-data that drupal 7.33+ generates in the database. See https://www.drupal.org/node/2381739